tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolBIRTest.java

```diff
@@ -390,4 +391,21 @@ public void testHasDefaultValue(int line, int col, Boolean hasDefault) {
             assertEquals((Boolean) fieldSymbol.hasDefaultValue(), hasDefault);
         }
     }
+
+    @Test
+    public void testSymbolPosBIR() {
+        Project project = BCompileUtil.loadProject("test-src/symbol_position_bir_test.bal");
+        Package currentPackage = project.currentPackage();
+        ModuleId defaultModuleId = currentPackage.getDefaultModule().moduleId();
+        Document srcFile = getDocumentForSingleSource(project);
+
+        PackageCompilation packageCompilation = currentPackage.getCompilation();
+        SemanticModel model = packageCompilation.getSemanticModel(defaultModuleId);
+        LineRange lineRange = ((TypeReferenceTypeSymbol) model.symbol(srcFile,
+                from(19, 16)).get()).definition().getLocation().get().lineRange();
+        assertEquals(lineRange.startLine().line(), 143);
```

Comment:
Shall we keep `143` as a variable and have line 408 as that var + `4` to make it easier for future changes to the file?

Reply:
mm. Good suggestion, but I think we don't need to do it in that way, Most of the Semantic API test cases in a way we compare exact values.